### PR TITLE
Bump station

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Please describe what the changes are made in this pull request and why the change was necessary.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: config/business_info.yml [hunk 1]
```diff
@@ -15,12 +16,9 @@ header:
     sign-up:
       path: https://dashboard.nexmo.com/sign-up
       text:
-        - '{ Try it free }'
-        - '< Try it free />'
-        - '{ Get an API key }'
-        - '< Get an API key />'
-        - '{ Start building for free }'
-        - '< Start building for free />'
+        - try-it
```
Comment (bencgreenberg) [inline config/business_info.yml:19]: Are we removing all the other texts here on purpose?

Comment (fabianrbz) [reply config/business_info.yml:19]: yeah, these are the keys for the actual texts in `config/locales/<language>.yml`

Comment (mheap) [reply config/business_info.yml:19]: This isn't a like for like change. Part of the A/B test is the surrounding characters

Comment (fabianrbz) [reply config/business_info.yml:19]: @mheap @bencgreenberg that's being handled in `station`

**Pull request merged**